performance-tracking/src/main/java/com/rakuten/tech/mobile/perf/runtime/internal/ConfigurationResult.java

```diff
@@ -22,14 +22,24 @@
   @SerializedName("sendHeaders")
   private Map<String, String> header;
 
+  @SerializedName("modules")
```

Comment:
looking at the code i don't think we really need Gson for anything in this code base, we barely use it but impose it on SDK users for no good reason....

Reply:
Yeah, I think it's only used for the results from the config and location APIs, so it could be removed pretty easily.